src/compiler/program.ts

```diff
@@ -1817,8 +1815,7 @@ namespace ts {
             collectExternalModuleReferences(file);
             if (file.imports.length || file.moduleAugmentations.length) {
                 // Because global augmentation doesn't have string literal name, we can check for global augmentation as such.
-                const nonGlobalAugmentation = filter(file.moduleAugmentations, (moduleAugmentation) => moduleAugmentation.kind === SyntaxKind.StringLiteral);
-                const moduleNames = map(concatenate(file.imports, nonGlobalAugmentation), getTextOfLiteral);
+                const moduleNames = getModuleImportAndAugmentationNames(file);

```

Comment:
Should the name of this local indicate that it contains augmentations?  (It may be implied - I don't know what an augmentation is.)

Reply:
A module augmentation adds new members to an external module. In order to do this we need to resolve that module, so we're considering both imports and augmentations together on purpose, since they both need to be resolved.
Handbook ref: http://www.typescriptlang.org/docs/handbook/declaration-merging.html

Reply:
It just seems strange that we say "ImportAndAugmentationNames" in the function name but just "moduleNames" in the local name.

Reply:
How about just calling the function `getModuleNames` then?

Reply:
If they're consistent, I'm happy. :)